tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -1191,26 +1360,28 @@ public function test_capture_charge_expired() {
 
 		$result = $this->wcpay_gateway->capture_charge( $order );
 
-		$note = wc_get_order_notes(
+		$notes             = wc_get_order_notes(
 			[
 				'order_id' => $order->get_id(),
-				'limit'    => 1,
+				'limit'    => 2,
 			]
-		)[0];
+		);
+		$latest_wcpay_note = $notes[1]; // The latest note is the "status changed" message, we want the previous one.
 
 		// Assert the returned data contains fields required by the REST endpoint.
 		$this->assertEquals(
 			[
-				'status'    => 'failed',
+				'status'    => 'succeeded',
 				'id'        => $intent_id,
-				'message'   => 'test exception',
-				'http_code' => 500,
+				'message'   => null,
+				'http_code' => 200,
 			],
 			$result
 		);
-		$this->assertContains( 'expired', $note->content );
-		$this->assertEquals( $order->get_meta( '_intention_status', true ), 'canceled' );
-		$this->assertEquals( $order->get_status(), 'cancelled' );
+		$this->assertContains( 'successfully captured', $latest_wcpay_note->content );
+		$this->assertContains( wc_price( $order->get_total() ), $latest_wcpay_note->content );
+		$this->assertEquals( $order->get_meta( '_intention_status', true ), 'succeeded' );
```

Comment:
`assertSame` =)

Reply:
Gosh, how did I miss this despite constant prodding from you 😄  
Done ✅